tests/console/ncurses.pm

```diff
@@ -24,7 +24,11 @@ sub run {
     select_console 'root-console';
     zypper_call 'in dialog';
     script_run 'dialog --yesno "test for boo#1054448" 3 20', 0;
-    assert_screen 'ncurses-simple-dialog';
+    assert_screen([qw(ncurses-simple-dialog ncurses-simple-dialog-broken-poo63026)]);
+    if (match_has_tag 'ncurses-simple-dialog-broken-poo63026') {
+        # poo#63026: Sometimes the console is misconfigured in openQA and shows a broken UI
+        die "Console misconfigured, ncurses UI broken!\n";
```

Comment:
I suggest to just combine the comment and die message. With the ticket reference one can automatically follow the issue tracker for more information

Reply:
Fixed